documentation/using_plugins.md

```diff
@@ -197,6 +197,8 @@ plugin-hooks:
       action: install
 ```
 
+**NOTE**: If you are configuring plugin hooks in an inventory file (see [inventory.yaml reference](bolt_inventory_reference.md#plugin_hooks)) you must use `plugin_hooks` instead of `plugin-hooks`.
+
```

Comment:
```suggestion
**NOTE**: If you are configuring plugin hooks in an inventory file, you must use `plugin_hooks`
instead of `plugin-hooks`. For more information, see the [inventory.yaml reference]
(bolt_inventory_reference.md#plugin_hooks).
```